examples/persistent-volumes/volumes/local-01.yaml

```diff
@@ -0,0 +1,11 @@
+kind: PersistentVolume
+apiVersion: v1beta3
+metadata:
+  name: pv0001
+  labels:
+    type: local
```

Comment:
Oh, and Mark, please excuse the very late feedback, and somewhat
unstructured nature of it.  I'm just going through all of the storage stuff
as best I can to get up to speed, making comments/notes as I go.  Feel free
to ignore all of my input for now, and I can provide consolidated feedback
at the end, if that works better for you.

On Thu, Mar 19, 2015 at 11:40 AM, Mark Turansky <notifications@github.com>
wrote:

> In examples/persistent-volumes/volumes/local-01.yaml
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5105#discussion_r26784869>
> :
>
> > @@ -0,0 +1,11 @@
> > +kind: PersistentVolume
> > +apiVersion: v1beta3
> > +metadata:
> > +  name: pv0001
> > +  labels:
> > +    type: local
>
> Agreed, this has caused confusion already in reviewers. I added comments
> that specifically state that a hostPath volume should be used only and
> exclusively for development and testing.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/GoogleCloudPlatform/kubernetes/pull/5105/files#r26784869>
> .
>